docs/next/guides/internationalization/layout-direction.md

```diff
@@ -0,0 +1,222 @@
+| [Cell types](@/guides/cell-types/cell-type.md)                       | [`autocomplete`](@/guides/cell-types/autocomplete-cell-type.md)<br>[`checkbox`](@/guides/cell-types/checkbox-cell-type.md)<br>[`date`](@/guides/cell-types/date-cell-type.md)<br>[`dropdown`](@/guides/cell-types/dropdown-cell-type.md)<br>[`handsontable`](@/guides/cell-types/handsontable-cell-type.md)<br>[`numeric`](@/guides/cell-types/numeric-cell-type.md)<br>[`time`](@/guides/cell-types/time-cell-type.md)                                                                                                                                                       |
+| [Configuration options](@/guides/getting-started/setting-options.md) | [`colHeaders`](@/api/options.md#colheaders)<br>[`fixedColumnsLeft`](@/api/options.md#fixedcolumnsleft)<br>[`fixedRowsBottom`](@/api/options.md#fixedrowsbottom)<br>[`rowHeaders`](@/api/options.md#rowheaders)<br>[`selectionMode`](@/api/options.md#selectionmode)<br>                                                                                                                                                                                                                                                                                                       |
+| [Plugins](@/api/plugins.md)                                          | [`Autofill`](@/api/autofill.md)<br>[`CollapsibleColumns`](@/api/collapsiblecolumns.md)<br>[`ColumnSorting`](@/api/columnsorting.md)<br>[`Comments`](@/api/comments.md)<br>[`ContextMenu`](@/api/contextmenu.md)<br>[`CustomBorders`](@/api/customborders.md)<br>[`DropdownMenu`](@/api/dropdownmenu.md)<br>[`Filters`](@/api/filters.md)<br>[`HiddenColumns`](@/api/hiddencolumns.md)<br>[`HiddenRows`](@/api/hiddenrows.md)<br>[`ManualColumnMove`](@/api/manualcolumnmove.md)<br>[`ManualColumnResize`](@/api/manualcolumnresize.md)<br>[`NestedRows`](@/api/nestedrows.md) |
+| Other                                                                | Horizontal scroll<br>[Keyboard navigation](@/guides/accessories-and-menus/keyboard-navigation.md) (<kbd>Tab</kbd>)<br>[Selection](@/guides/cell-features/selection.md)<br>[UI indicators](@/guides/columns/column-hiding.md#step-2-show-ui-indicators)<br>Vertical scroll                                                                                                                                                                                                                                                                                                     |
```

Comment:
Please make sure to mention the different direction of the Tab and Tab+Shift on the `keyboard-navigation.md` page.

Reply:
Implemented in 8581f60b6fa51bc9be71332dcdfee9606f24df7d